tasks/rebuild/create_archives.js

```diff
@@ -0,0 +1,23 @@
+import { execFileSync } from 'child_process';
+import { join } from 'path';
+
+export default (grunt) => {
+  grunt.registerTask('_rebuild:createArchives', function () {
```

Comment:
I originally tried to go down that route, but it would have required pretty fundamental changes to `_build:archives`, so I decided against it.  Basically, `_build:archives` assumes that all platforms are present in the build directory (something not guaranteed in a rebuild) and that all builds have the same version as the current working copy's package.json (something not true if you're using the `buildversion` argument).